test/test_example_parsers/test_dice_roll_parser.py

```diff
@@ -63,3 +63,9 @@ def test_compound_examples(self, die,
                                evaluation_string,
                                result_min, result_max):
         assert result_min <= die.evaluate(evaluation_string) <= result_max
+
+    def test_rolls_are_distinct(self, die):
+        pairs_of_rolls = []
+        for pair in range(100):
+            pairs_of_rolls.append(die.evaluate('2d6'))
```

Comment:
List comprehension?
22 should be 2
Add error message for asserrts

     pairs_of_rolls = [die.evaluate('2d6') for _ in range(100)]
     assert any(pair % 2 != 0 for pair in pairs_of_rolls), "incorrect evaluation of multiple die rolls"

Reply:
Gah. I retyped after shortening the var.

Reply:
You're right, a list comp is better. Actually, you could lose the comparison, if you wanted, and just have `pair%2 for pair in pairs of rolls` - but I don't think that's as clear to read.